packages/@coorpacademy-player-store/src/actions/ui/progressions.js

```diff
@@ -62,6 +62,14 @@ export const unselectProgression: SelectAction = {
   }
 };
 
+const fetchData = (dispatch, engine, progressionId, progressionContent): Promise<any> =>
```

Comment:
Can this one have the common actions signature ? to be able to call it elsewhere than there?

Reply:
ok why not, but it's sure it won't be reused elsewhere :) 
https://github.com/CoorpAcademy/components/pull/1618/commits/7d7752d409fb1f233e497b0da0ce2ac414b743f3